src/main/java/net/minecraftforge/fml/config/ConfigFileTypeHandler.java

```diff
@@ -91,6 +93,35 @@ private boolean setupConfigFile(final ModConfig modConfig, final Path file, fina
         return true;
     }
 
+    public static void backUpConfig(final CommentedFileConfig commentedFileConfig) {
+        int maxBackups = 5;
```

Comment:
Just asking, wouldn't this be better as a static constant, instead of a local constant?

Reply:
Added an overload that allows the caller to choose maxBackups and a TODO requesting ideas on how to allow mods to set maxBackups for their own config